common/settings.h

```diff
@@ -224,6 +224,10 @@ struct Settings {
   // must be available to the application.
   bool enable_vulkan_validation = false;
 
+  // Enable the implicit view, so that the shell should create a view with
+  // kFlutterImplicitViewId without the platform calling FlutterEngineAddView.
+  bool enable_implicit_view = true;
```

Comment:
Can we give a brief explanation on what the implicit view is (has view ID 0, allows app to make single view assumptions, used for mobile apps) and mention that `PlatformDispatcher.implicitView` will be non-null if this is true?

Reply:
I wrote some doc. Let me know what you think :)

Reply:
Excellent comment. Marking this as resolved :)